src/filters/dasher.c

```diff
@@ -10070,11 +10070,13 @@ static void dasher_process_hls_ll(GF_DasherCtx *ctx, const GF_FilterEvent *evt)
 		GF_LOG(GF_LOG_ERROR, GF_LOG_DASH, ("[Dasher] Received segment size info event but no pending segments\n"));
 		return;
 	}
-	sctx->frags = gf_realloc(sctx->frags, sizeof (GF_DASH_FragmentContext) * (sctx->nb_frags+1));
-	if (!sctx->frags) {
+	void *new_frags = gf_realloc(sctx->frags, sizeof (GF_DASH_FragmentContext) * (sctx->nb_frags+1));
+	if (!new_frags) {
+		gf_free(sctx->frags);
```

Comment:
Hi, 

Looks good, just a small thing: when we free a pointer inside a structure, it's good practice to force-set it to null to avoid double-frees and such things later on.

Reply:
Thank you for the tip